yarn.lock

```diff
@@ -15046,7 +15051,7 @@ metal-keyboard-focus@^2.0.0:
     metal "^2.0.0"
     metal-events "^2.2.3"
 
-metal-position@^2.1.0, metal-position@^2.1.2:
+metal-position@^2.1.0:
```

Comment:
note that metal-position is no longer a dependency for clay v3, but it is still used on clayui.com via tooltip v2